packages/esm-patient-list-app/src/patient-list-list/patient-list-table.component.tsx

```diff
@@ -85,11 +85,6 @@ const PatientListTableContainer: React.FC<PatientListTableContainerProps> = ({
   }
   const { paginated, goTo, results, currentPage } = usePagination(sortedData, pageSize);
 
-  const handleSearch = (str) => {
-    setSearchString(str);
-    goTo(1);
-  };
-
```

Comment:
Revert this function, since we need to run `goTo(1)` because of the following scenario:

When user opens the table and say goes to the 3rd page. The the user starts typing a search term, see the page with which the results appear is still 3. We need to redirect the user back to 1st page whenever user searches.

You can run this scenario on your own.

Instead of naming the prop `handleSearch`, let it be `setSearchTerm` and have this function working properly.

Reply:
Thanks alot @vasharma05, I tried to figure out what that was doing.